SignalServiceKit/src/Account/TSPreKeyManager.m

```diff
@@ -45,8 +50,53 @@
 
 #pragma mark -
 
+@interface TSPreKeyManager ()
+
+// This property should only be accessed on serialQueue.
+@property (atomic) NSDate *lastPreKeyCheckTimestamp;
+
+@end
+
+#pragma mark -
+
 @implementation TSPreKeyManager
 
++ (instancetype)sharedManager
```

Comment:
I made this a singleton to be consistent and so that we could enable NSNotifications.